windows-installer/build-installer.sh

```diff
@@ -102,15 +101,9 @@ tar -C tmp/jre/eclipse/jre -xaf packages/jre-*-windows-x64.tar.gz --strip-compon
 # call eclipse.exe instead of eclipsec.exe and that seemed to solve the
```

Comment:
This comment was related to `eclipsec.exe` invocation, therefore it can now be removed.